terraform-infra-approvals/pre-shared-infrastructure.json

```diff
@@ -16,10 +16,12 @@
     {"type": "azurerm_virtual_network_peering"},
     {"type": "azurerm_media_transform"},
     {"type": "azurerm_role_assignment"},
-    {"type": "azurerm_disk_encryption_set"}, 
-    {"type": "azurerm_key_vault_key"}
-    
+    {"type": "azurerm_disk_encryption_set"},
+    {"type": "azurerm_key_vault_key"},
+    {"type": "azurerm_monitor_diagnostic_setting"}
  
   ],
-  "module_calls": []
+  "module_calls": [
+    {"source":  "git@github.com:hmcts/terraform-module-log-analytics-workspace-id.git?ref=master"}
+  ]
```

Comment:
you do not need this

Reply:
yes, for the diagnostic settings to be able to write to the  log analytics using existing module

Reply:
ah right mis-read that